README.md

```diff
@@ -15,22 +15,31 @@
 [![code style black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![codecov](https://codecov.io/gh/cmudig/draco2/branch/main/graph/badge.svg)](https://codecov.io/gh/cmudig/draco2)
 [![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/cmudig/draco2.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/cmudig/draco2/context:python)
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/cmudig/draco2/HEAD)
 
 **Work in Progress**
 
-Draco is a formal framework for representing design knowledge about effective visualization design as a collection of constraints. You can use Draco to find effective visualization visual designs or validate visualization designs. Draco's constraints are based on Answer Set Programming (ASP) and solved with the [Clingo](https://github.com/potassco/clingo) constraint solver. We also implemented a way to learn weights for the recommendation system directly from the results of graphical perception experiment. Draco v2 is a much improved version of the first iteration of [Draco](https://github.com/uwdata/draco).
+Draco is a formal framework for representing design knowledge about effective visualization design as a collection of
```

Comment:
Is this formatting something we can do automatically in e.g. a hook? Otherwise I suspect the formatting will get worse if someone makes an edit in the future.

Reply:
I added a new entry to our pre-commit config & opened a separate PR for it (#396)